decision_service/include/str_util.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <string>
+#include <sstream>
+
+namespace reinforcement_learning { namespace utility {
+
+  template <typename Last>
+  std::string concat(std::ostringstream& os, const Last& last) {
```

Comment:
Where is this actually used?

Reply:
This is used in config_utility.cc.  I added a pointer on the relevant line.

Reply:
This doesn't matter to much, but for utility functions only used in one place it seems stylistically better to have it defined in that place.